include/ddc/kernels/splines/matrix.hpp

```diff
@@ -69,6 +108,14 @@ class Matrix
         return b;
     }
 
+    /**
+     * @brief Solve the multiple-right-hand-side linear problem Ax=b inplace.
+     *
+     * @param[in, out] bx A 2D mdpsan storing the multiple right-hand-sides of the problem and receiving the corresponding solution. Important note: the convention is the reverse of the common matrix one,
+     * row number is second index and column number the first one. It should be changed in the future.
```

Comment:
```suggestion
     * @param[in, out] bx A 2D mdpsan storing the multiple right-hand-sides of the problem and receiving the corresponding solution.
     * Important note: the convention is the reverse of the common matrix one,
     * row number is second index and column number the first one. It should be changed in the future.
```
Long lines in these files

Reply:
![image](https://github.com/CExA-project/ddc/assets/25363473/9cf43883-7869-4393-94c6-bcfae4840e0c)

This is not much shorter. For reference the advised 80 character limit is here:
![image](https://github.com/CExA-project/ddc/assets/25363473/e324808f-a9dd-4920-b921-51f8561b18df)

Reply:
Yes sry, I do it "by eye" and I am on a bigger screen so it was longer :/